website/static/js/share/utils.js

```diff
@@ -30,6 +33,7 @@ utils.formatNumber = function(num) {
 
 var loadingIcon = m('img[src=/static/img/loading.gif]',{style: {margin: 'auto', display: 'block'}});
```

Comment:
hmm... Saw it was only used in one place, but I made it a component and used it in that one place. For some reason, the image is no longer displaying. Haven't been able to see a reason why yet...
```
utils.loadingIcon = {
    view: function(ctrl) {
        return m('img[src=/static/img/loading.gif]',{style: {margin: 'auto', display: 'block'}});
    }
};
...
m('', [m.component(utils.loadingIcon), 'loading...']);
```